iOS/dough-bros-ios/Model/User.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  User.swift
+//  dough-bros-ios
+//
+//  Created by Harin Wu on 2020-11-10.
+//
+
+import Foundation
+
+public class User {
```

Comment:
For data models, you can use structs instead of classes. That way you don't have to define your own constructors. Also, if you need to decode/encode from JSON, you can just conform to the "Codable" protocol and specify CodingKeys